### DISCUSSION TITLE
Using Promitor to monitor Virtual Machine disks

### DISCUSSION THREAD

**Discussion by deggja** [tomkerkhove/promitor#2159] [Q&A] [answered]
Hey guys,

I'm a newbie but I have recently started using Promitor together with Prometheus to monitor resources running in Azure. I am currently trying to figure out how to monitor available disk space and availability of a virtual machine in Azure using Promitor. As far as I can see, there's no exposed Azure Monitor data for these particular metrics according to the docs - so it might not even be possible.

Did anyone here face this issue before & if so - how did you deal with it?

Thanks!

**Comment by tomkerkhove** [comment, marked as answer]
I'm afraid only these metrics are available OOTB:
- https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/metrics-supported#microsoftcomputedisks
- https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/metrics-supported#microsoftcomputevirtualmachines
